yt_astro_analysis/cosmological_observation/light_cone/light_cone.py

```diff
@@ -335,14 +335,9 @@ def project_light_cone(self, field_of_view, image_resolution, field,
                 pixel_area = (proper_box_size.in_cgs() / pixels)**2 #in proper cm^2
                 factor = pixel_area / (4.0 * np.pi * dL.in_cgs()**2)
                 mylog.info("Distance to slice = %s" % dL)
-                frb[field] *= factor #in erg/s/cm^2/Hz on observer"s image plane.
+                frb['field'] *= factor #in erg/s/cm^2/Hz on observer"s image plane.
```

Comment:
As long as you're changing this line, there is a double quote where a single would do

Reply:
Oh yeah, that's probably been there forever!